src/timeline.js

```diff
@@ -78,7 +86,10 @@ export default class Timeline extends React.Component {
     onItemHover: PropTypes.func,
     onItemLeave: PropTypes.func,
     itemRenderer: PropTypes.func,
+    // The renderer of the columns. The DefaultGroupRenderer has a getTitle() function that returns group.title
+    // if the dataKey is empty, otherwise it returns group[dataKey]. This function can be overriden.
     groupRenderer: PropTypes.func,
```

Comment:
I'd put this comment:

Single column mode: the renderer of a cell. 
Multiple columns mode: the default renderer of a cell, which may be overridden on a per column basis.